src/mvs/patch_match.cc

```diff
@@ -153,31 +158,10 @@ void PatchMatch::Check() const {
   }
 }
 
-void PatchMatch::Run() {
-  PrintHeading2("PatchMatch::Run");
-
-  Check();
-
-  patch_match_cuda_.reset(new PatchMatchCuda(options_, problem_));
```

Comment:
Removed this in favor of an inheritance structure where `PatchMatchCuda` and the new `PatchMatchNet` both inherit from `PatchMatch`. Not sure what was the intent of using `PatchMatchCuda` inside a `unique_ptr` so if that breaks something important I could change it back.